src/usr/lib/rsetup/tui/task/baota/baota.sh

```diff
@@ -0,0 +1,34 @@
+# shellcheck shell=bash
+
+__task_baota_uninstall() {
+    if yesno "Are you sure to uninstall Baota?"
+    then
+        if uninstall_baota
+        then
+            msgbox "Uninstall Baota success."
+        else
+            msgbox "Uninstall Baota failure."
```

Comment:
```suggestion
            msgbox "An failure occurred when trying to uninstall Baota."  "$RTUI_PALETTE_ERROR"
```
$RTUI_PALETTE_ERROR can be used to display a red error message box.